src/contracts/Chef.sol

```diff
@@ -1,203 +1,45 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/access/AccessControl.sol";
-import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 import "./MetricToken.sol";
 
-// Heavily Inspired by Sushi's MasterChefv2 - but with a few changes:
```

Comment:
can you move this to a readme?  the wall of text was how I worked out how it works and might be useful for future contributors (agree it shouldn't be in the file)